README.md

```diff
@@ -19,8 +36,10 @@ Commands:
 python3 -m venv venv
 source ./venv/bin/activate
 pip install -r requirements.txt
+mkdir -p build/data
+python3 download_data.py build/data
 python3 generate_sites.py
-python3 -m http.server
+python3 -m http.server 8000 --directory build
```

Comment:
Independent of getting this pull request in, the readme instructions for how to run this were missing a few steps and argments. I found this out by setting up these steps for the self-contained docker image.

Reply:
Part of these changes are already merged 59861cdb3b6a8c801e9ba8b6cd377f7671d73b4b